lms/djangoapps/teams/views.py

```diff
@@ -541,6 +561,15 @@ def get_queryset(self):
         """Returns the queryset used to access the given team."""
         return CourseTeam.objects.all()
 
+    def delete(self, request, team_id):
+        """DELETE /api/team/v0/teams/{team_id}"""
+        team = get_object_or_404(CourseTeam, team_id=team_id)
+        self.check_object_permissions(request, team)
```

Comment:
Yeah, if I remove that line the tests around permissions to delete fail. https://tomchristie.github.io/rest-framework-2-docs/api-guide/permissions